src/constants/task.ts

```diff
@@ -0,0 +1,14 @@
+export const EMPTY_TASKS_DATA = {
+    ALL: [],
+    IN_PROGRESS: [],
+    ASSIGNED: [],
+    AVAILABLE: [],
+    UNASSIGNED: [],
+    NEEDS_REVIEW: [],
+    IN_REVIEW: [],
+    VERIFIED: [],
+    MERGED: [],
+    COMPLETED: [],
+    OVERDUE: [],
+    DONE: [],
+};
```

Comment:
This above is already in https://github.com/Real-Dev-Squad/website-status/blob/develop/src/constants/tasks.ts#L1

Reply:
@shubhamsigdar1, Yes,This is removed now as this was added to resolve conflicts in the early stages